azure-pipelines.yml

```diff
@@ -0,0 +1,47 @@
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  PYTHON_VERSION: 3.6
+
```

Comment:
On Windows, the python version must be> = 3.6 to install [atari-py](https://pypi.org/project/atari-py/#files).